cypress.json

```diff
@@ -1,5 +1,6 @@
 {
   "baseUrl": "http://localhost:3000",
   "defaultCommandTimeout": 10000,
-  "projectId": "m7sj8t"
+  "projectId": "m7sj8t",
+  "pageLoadTimeout": 120000
```

Comment:
This seems to make it pass in CI.  The default timeout is 60s and apparently we're going over for some reason (I assume this is basically the time it takes for 'yarn start' to complete but not sure).